site/content/docs/_index.md

```diff
@@ -0,0 +1,164 @@
++++
+title = "Docs"
+layout = "single"
++++
+
+
+## API
```

Comment:
I think overview will be better before anything else. Otherwise, it looks weird to enter with a link like this. I can supply some boilerplate overview if you want.